package.json

```diff
@@ -41,7 +41,7 @@
   "homepage": "https://github.com/airbnb/react-with-styles-interface-aphrodite#readme",
   "devDependencies": {
     "airbnb-js-shims": "^1.4.0",
-    "aphrodite": "^1.2.5",
+    "aphrodite": "^2.1.0",
```

Comment:
Are these changes backwards compatible with aphrodite down to 0.5.0?

Regardless, we should change the peer dep to not be just `>=`; but it'd be ideal if we could do things in a backwards compatible way.

Reply:
They are not backwards compatible.